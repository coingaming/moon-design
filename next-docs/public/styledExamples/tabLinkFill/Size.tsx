import { TabLinkFill } from '@heathmont/moon-core';

const Example = () => <TabLinkFill size="small" />;

export default Example;
