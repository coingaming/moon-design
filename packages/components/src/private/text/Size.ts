export type Size = 10 | 12 | 14 | 16 | 18 | 20 | 24 | 32 | 48 | 56 | 64 | 72;
