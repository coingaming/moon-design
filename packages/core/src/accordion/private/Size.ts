type Size = 'small' | 'medium' | 'large' | 'xlarge';

export default Size;
