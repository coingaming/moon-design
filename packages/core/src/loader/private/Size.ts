export type Size = 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge';
