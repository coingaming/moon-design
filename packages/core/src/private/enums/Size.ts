enum Size {
  TWOXSMALL = '2xsmall',
  XSMALL = 'xsmall',
  SMALL = 'small',
  MEDIUM = 'medium',
  LARGE = 'large',
  XLARGE = 'xlarge',
  TWOXLARGE = '2xlarge',
}

export default Size;
