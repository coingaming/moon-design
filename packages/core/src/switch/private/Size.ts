export type Size = '2xs' | 'xs' | 'sm' | 'md' | 'lg';
