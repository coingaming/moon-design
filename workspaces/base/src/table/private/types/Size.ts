type Size = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | '2xl';

export default Size;
