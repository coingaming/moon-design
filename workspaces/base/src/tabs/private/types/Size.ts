type Size = 'sm' | 'md';

export default Size;
