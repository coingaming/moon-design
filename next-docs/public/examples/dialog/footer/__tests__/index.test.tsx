/**
 * @jest-environment jsdom
 */

import React from 'react';
import { moonDesignLight, ThemeProvider } from '@heathmont/moon-themes';
import renderer from 'react-test-renderer';
import Size from '../Size';
import ButtonSm from '../ButtonSm';
import ButtonXs from '../ButtonXs';

const renderWithTheme = (component: JSX.Element) => (
  <ThemeProvider theme={moonDesignLight}>{component}</ThemeProvider>
);

describe('Dialog Footer', () => {
  it('renders different Sizes', () => {
    const testRenderer = renderer.create(renderWithTheme(<Size />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders with Button XS', () => {
    const testRenderer = renderer.create(renderWithTheme(<ButtonXs />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders with Button SM', () => {
    const testRenderer = renderer.create(renderWithTheme(<ButtonSm />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
