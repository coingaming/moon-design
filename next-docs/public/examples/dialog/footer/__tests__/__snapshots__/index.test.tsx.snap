// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog Footer renders different Sizes 1`] = `
<div
  className="flex flex-col"
>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 esdulX"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ jgyICO"
            size="small"
            variant="primary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ jJFhAA"
            size="small"
            variant="tertiary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
      <div
        className="ButtonWrapper-k744ak-0 dvmfaN"
      >
        <button
          className="sc-jEUrFJ jsVuhc"
          size="small"
          variant="secondary"
        >
          <div
            className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
          >
            Button
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kELTOD"
      size="small"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ idkWst"
            size="xsmall"
            variant="primary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ eqpIFL"
            size="xsmall"
            variant="tertiary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
      <div
        className="ButtonWrapper-k744ak-0 dvmfaN"
      >
        <button
          className="sc-jEUrFJ cIanBH"
          size="xsmall"
          variant="secondary"
        >
          <div
            className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
          >
            Button
          </div>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Dialog Footer renders with Button SM 1`] = `
<div
  className="flex flex-col"
>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ jgyICO"
            size="small"
            variant="primary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ jgyICO"
            size="small"
            variant="primary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ jJFhAA"
            size="small"
            variant="tertiary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ jgyICO"
            size="small"
            variant="primary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ jJFhAA"
            size="small"
            variant="tertiary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
      <div
        className="ButtonWrapper-k744ak-0 dvmfaN"
      >
        <button
          className="sc-jEUrFJ jsVuhc"
          size="small"
          variant="secondary"
        >
          <div
            className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
          >
            Button
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ jgyICO"
            size="small"
            variant="primary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
      <div
        className="ButtonWrapper-k744ak-0 dvmfaN"
      >
        <button
          className="sc-jEUrFJ jsVuhc"
          size="small"
          variant="secondary"
        >
          <div
            className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
          >
            Button
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ jJFhAA"
            size="small"
            variant="tertiary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ jJFhAA"
            size="small"
            variant="tertiary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
      <div
        className="ButtonWrapper-k744ak-0 dvmfaN"
      >
        <button
          className="sc-jEUrFJ jsVuhc"
          size="small"
          variant="secondary"
        >
          <div
            className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
          >
            Button
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-full max-w-screen-sm mb-4"
  />
</div>
`;

exports[`Dialog Footer renders with Button XS 1`] = `
<div
  className="flex flex-col"
>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ idkWst"
            size="xsmall"
            variant="primary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ idkWst"
            size="xsmall"
            variant="primary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ eqpIFL"
            size="xsmall"
            variant="tertiary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ idkWst"
            size="xsmall"
            variant="primary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ eqpIFL"
            size="xsmall"
            variant="tertiary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
      <div
        className="ButtonWrapper-k744ak-0 dvmfaN"
      >
        <button
          className="sc-jEUrFJ cIanBH"
          size="xsmall"
          variant="secondary"
        >
          <div
            className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
          >
            Button
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ idkWst"
            size="xsmall"
            variant="primary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
      <div
        className="ButtonWrapper-k744ak-0 dvmfaN"
      >
        <button
          className="sc-jEUrFJ cIanBH"
          size="xsmall"
          variant="secondary"
        >
          <div
            className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
          >
            Button
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ eqpIFL"
            size="xsmall"
            variant="tertiary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-96 mb-4"
  >
    <div
      className="Container-srtaaf-0 kFekyR"
    >
      <div
        className="RightWrapper-vueweo-0 ljvpRC"
      >
        <div
          className="ButtonWrapper-k744ak-0 dvmfaN"
        >
          <button
            className="sc-jEUrFJ eqpIFL"
            size="xsmall"
            variant="tertiary"
          >
            <div
              className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
            >
              Button
            </div>
          </button>
        </div>
      </div>
      <div
        className="ButtonWrapper-k744ak-0 dvmfaN"
      >
        <button
          className="sc-jEUrFJ cIanBH"
          size="xsmall"
          variant="secondary"
        >
          <div
            className="Button__InnerContainer-sc-1xtdszg-0 gwKSbc"
          >
            Button
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="flex justify-around items-center w-full max-w-screen-sm mb-4"
  />
</div>
`;
